actions/pom.xml

```diff
@@ -26,6 +26,11 @@
       <groupId>io.cloudevents</groupId>
       <artifactId>cloudevents-core</artifactId>
     </dependency>
+    <dependency>
```

Comment:
See my question on whether we can (and should) use VertX rather than introducing a dependency on another HTTP client.

Reply:
Same comment as [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/128#discussion_r750984055)

Reply:
Switched to VertX WebClient :+1: